packages/f-content-cards/src/components/cardTemplates/_tests/TermsAndConditionsCard.test.js

```diff
@@ -0,0 +1,74 @@
+/* eslint indent: ["error", 4, {ignoredNodes: ["TemplateLiteral > *"]}] */
+import { createLocalVue, shallowMount } from '@vue/test-utils';
+import TermsAndConditionsCard from '../TermsAndConditionsCard.vue';
+
+const localVue = createLocalVue();
+
+jest.mock('copy-to-clipboard');
+
+const ctaText = '__CTA_TEXT__';
+const url = 'https://foo.com/bar';
+const subtitle = '__SUBTITLE__';
+const title = '__TITLE__';
+
+const card = (overrides = {}) => ({
+    ...{
+        title,
+        subtitle,
+        ctaText,
+        url
+    },
+    ...overrides
```

Comment:
Apologies if I've missed something critical but can you not change this over to:

```suggestion
    title,
    subtitle,
    ctaText,
    url,
    ...overrides
```

Reply:
Indeed I can! Think I was probably thinking along the lines of grouping all those potentially overridable keys together to make it easy to make the mental distinction between those and the object that then overrides them, but happy to change to remove the curlies.